storybook/public/rheostat.css

```diff
@@ -21,7 +21,7 @@
 }
 
 .rheostat-progress {
-  background-color: #00a699;
+  background-color: #3369e7;
```

Comment:
No but I spot it by doing this renaming (It's to align the rheostat color with our default theme color).